Library/ViewModels/CreatePasswordViewModelTests.swift

```diff
@@ -39,19 +47,47 @@ final class CreatePasswordViewModelTests: TestCase {
   }
 
   func testCreatePassword() {
+    let service = MockService()
```

Comment:
Do we need a local variable since we're not reusing it later on?

Reply:
Nope, removed it!